x/sysproxy/sysproxy_test.go

```diff
@@ -0,0 +1,107 @@
+// go:build (linux && !android) || windows || darwin
```

Comment:
```suggestion
// Copyright 2024 Jigsaw Operations LLC
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//     https://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

// go:build (linux && !android) || windows || darwin
```

Reply:
You may need `(darwin && ! ios)` as well